packages/server/lib/browsers/firefox-util.ts

```diff
@@ -135,7 +135,7 @@ async function connectToNewSpec (options, automation: Automation, browserCriClie
 }
 
 async function setupRemote (remotePort, automation, onError, options): Promise<BrowserCriClient> {
-  const browserCriClient = await BrowserCriClient.create(remotePort, 'Firefox', onError)
+  const browserCriClient = await BrowserCriClient.create(['127.0.0.1', '::1'], remotePort, 'Firefox', onError)
```

Comment:
Does this fix also need to apply to the Marionette or Foxdriver connections?

Reply:
I didn't observe any connection issues for Marionette or Foxdriver as they both were started on 127.0.0.1.

```
  cypress:network:connect successfully connected { opts: { host: '127.0.0.1', port: 64061, getDelayMsForRetry: [Function: getDelayMsForRetry] }, iteration: 2 } +103ms
  cypress:server:browsers:firefox-util foxdriver: { host: '127.0.0.1', port: 64061 } +0ms
  cypress:network:connect successfully connected { opts: { host: '127.0.0.1', port: 64062, getDelayMsForRetry: [Function: getDelayMsForRetry] }, iteration: 2 } +1ms
  cypress:server:browsers:firefox-util marionette: { host: '127.0.0.1', port: 64062 } +1ms
```